.github/workflows/ci.yml

```
@@ -52,7 +52,23 @@ jobs:
           - send
           - getMetadata
           - governance
+          - ethBridge
     steps:
+    - name: start local Eth node setup
+      uses: actions/checkout@v2
+      with:
+        repository: cennznet/bridge-contracts
+        ref: main
+    - name: install
+      run: yarn
+    - name: build
+      run: yarn build
+    - name: start local Eth node
+      run: |
+        docker run --dns=127.0.0.1 -dit --name ethNode -p 8545:8545 -p 30303:30303  ethereumoptimism/hardhat
```

```suggestion
        docker run --dns=127.0.0.1 -d --name ethNode -p 8545:8545 -p 30303:30303  ethereumoptimism/hardhat
```

don't need `it` afaik

---

updated